package.json

```diff
@@ -52,6 +52,7 @@
     "react-addons-test-utils": "^15.0.2",
     "react-dom": "^15.0.2",
     "react-motion": "^0.4.2",
+    "react-router": "^2.0.1",
```

Comment:
I think this belongs in peerDependencies, otherwise it might conflict with the hosting project's version of react-router